modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,205 @@
+
+* For improved compliance with the OpenCloudMesh protocol specification (Federation) a https://github.com/owncloud/core/pull/35122[migration step] will convert the fields of the `remoteId` column of the `federated_reshares` and `share_external` tables from `int` to `string`. 
+  This migration might increase the upgrade duration depending on the number of federated shares.
+* A https://github.com/owncloud/core/pull/35721[repair step] has been added that drops the deprecated `contacts_cards_properties` table. 
+  This migration is not expected to increase the upgrade duration significantly.
+* A housekeeping https://github.com/owncloud/core/pull/36084[repair step] for the `oc_properties` table removes existing entries which have `fileid` with value `null` and restrict the further creation of such. 
+  This repair step is not expected to increase the upgrade duration significantly.
+
+=== Official PHP 7.3 support
+
+ownCloud Server 10.3 officially supports PHP 7.3. 
+The Server Core and all apps maintained by ownCloud have received a full QA cycle and are proven to work reliably with PHP 7.3.
+If you are still using version 5.6, you must upgrade PHP before upgrading ownCloud Server https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[as it's not supported anymore since ownCloud Server 10.2]. 
+If you are still running PHP 7.0 or 7.1, please plan an upgrade soon as these versions https://www.php.net/supported-versions.php[are or will soon be unsupported], respectively.
+See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud Documentation] for more information.
+
+=== PHP 7.0 deprecation note
+
+As announced with ownCloud Server https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation-2[10.0.8] and https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[10.2.0], support for PHP 7.0 is discontinued. 
+The next minor version of ownCloud Server (around the end of 2019) **no longer supports PHP 7.0**. 
+If you are still running on PHP 7.0, please make sure to plan an upgrade to PHP >= 7.2 to stay compatible.
+
+=== Changes to background job execution
+
+For code cleanup reasons, the execution of background jobs (e.g., for public link expiration, trash bin emptying, cleanup of old file versions) has been changed.
+
+**These changes require manual interaction in your installation:**
+
+* If you're using https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html#cron[System cron] to trigger background job execution, there is a new `occ` command (`occ system:cron`) which executes the background jobs. 
+  To make use of it, you have to change the entry in `crontab`. 
+  Instead of executing `cron.php` (e.g., `/usr/bin/php -f /path/to/your/owncloud/cron.php`), cron should use `occ system:cron` (e.g., `/usr/bin/php -f /path/to/your/owncloud/occ system:cron`). 
```

Comment:
@pmaier1, I'm running "10.3.0 alpha2" on my testing box and don't see `system:cron`; however, I do see `background:cron`. Is this the command that you're referencing?